### PR TITLE
fix: receive all notifications from deleted appointment

### PR DESCRIPTION
**Pull request by dhavaldodiya**
- fix issue of receiving all the notifications from deleted appointments

### CODE REVIEW

File: src/types/appointment-reminder.ts [hunk 1]
```diff
@@ -92,4 +92,10 @@ export type ResourceType = {
 	isRecurrent: boolean;
 	participationStatus: string;
 	alarm: boolean;
+	calendar: {
+		id: string;
+		name: string;
+		color: unknown;
```
Comment (geeky-abhishek) [inline src/types/appointment-reminder.ts]: I think we can define the exact type for the color other than unknown

File: src/view/reminder/appointment-reminder.tsx [hunk 2]
```diff
@@ -86,7 +86,8 @@ const AppointmentReminder: FC<AppointmentReminderProps> = (): ReactElement => {
 					if (
 						val?.resource.alarm &&
 						val?.resource.alarmData?.[0]?.nextAlarm > reminderRange.start &&
-						val?.permission === true
+						val?.permission === true &&
+						val.resource.calendar.id !== FOLDERS.TRASH
```
Comment (geeky-abhishek) [inline src/view/reminder/appointment-reminder.tsx]: val?.resource?.calendar?.id will be more solid

**Review by nubsthead — APPROVED** [review]

**Review by nubsthead — APPROVED** [review]

File: src/commons/zimbra-standard-colors.ts [hunk 3]
```diff
@@ -3,13 +3,13 @@
  *
  * SPDX-License-Identifier: AGPL-3.0-only
  */
-export type ZimbraColor = {
+export type ZimbraColorType = {
 	color: string;
 	background: string;
 	label?: string;
 };
```
Comment (geeky-abhishek) [inline src/commons/zimbra-standard-colors.ts:11]: @zovomat @nubsthead  can we create a separate ticket for replacing all the instances of this ZIMBRA_STANDARD_COLORS with the one that is exported from SHELL ?

**Review by geeky-abhishek — APPROVED** [review]

**Review by giuliano176 — APPROVED** [review]

File: src/commons/zimbra-standard-colors.ts (hunk 3 above)
Comment (nubsthead) [reply src/commons/zimbra-standard-colors.ts:11]: It would be a great idea, try to ask in the channel about it

**Pull request merged**